cephfs/path.go

```diff
@@ -44,3 +44,68 @@ func (mount *MountInfo) RemoveDir(path string) error {
 	ret := C.ceph_rmdir(mount.mount, cPath)
 	return getError(ret)
 }
+
+// Link creates a new link to an existing file
+//
+// Implements:
+// int ceph_link (struct ceph_mount_info *cmount, const char *existing,
+//				  const char *newname);
+func (mount *MountInfo) Link(existing, newname string) error {
```

Comment:
Let's change `existing` to `oldname`  to more closely match the naming in use by Go's os module. It's also a bit more self-consistent.